src/middlewared/middlewared/plugins/vm/vms.py

```diff
@@ -101,6 +101,8 @@ async def do_create(self, data):
         `shutdown_timeout` indicates the time in seconds the system waits for the VM to cleanly shutdown. During system
         shutdown, if the VM hasn't exited after a hardware shutdown signal has been sent by the system within
         `shutdown_timeout` seconds, system initiates poweroff for the VM to stop it.
+
+        Specifying `devices` is deprecated and will be removed in coming future release.
```

Comment:
`SCALE Angelfish: Specifying `devices` is deprecated and will be removed in next major release.`


Also wonder if we should have a deprecation warning in logs everytime that is used.

Reply:
Logs are for us, i don't think we should have it in the logs ? Thoughts ?

Reply:
Or mostly us :p ?

Reply:
Do you have a better idea how people will realized an API call is deprecated? Only in docs doesn't sound too good.

Developers often look at middleware log when using the API, I would imagine.

Reply:
True, i'll make necessary changes. Thank you